packages/shortcuts-extension/test/components/ShortcutUI.spec.ts

```diff
@@ -297,5 +297,165 @@ describe('@jupyterlab/shortcut-extension', () => {
         expect(data.user.shortcuts).toHaveLength(1);
       });
     });
+
+    describe('#sortShortcuts()', () => {
+      let mockedFilteredShortcutList: IShortcutTarget[];
+      beforeEach(() => {
+        mockedFilteredShortcutList = [
+          {
+            id: '1',
+            label: 'Zebra',
+            command: 'Zebra',
+            selector: 'Zebra',
+            category: 'Zebra',
+            keybindings: [{ keys: ['Ctrl+Z', 'Z'], isDefault: false }],
+            args: undefined
+          },
+          {
+            id: '2',
+            label: 'Apple',
+            command: 'Apple',
+            selector: 'Apple',
+            category: 'Apple',
+            keybindings: [{ keys: ['Shift+A', 'A'], isDefault: true }],
+            args: undefined
+          },
+          {
+            id: '3',
+            label: 'Banana',
+            command: 'Banana',
+            selector: 'Banana',
+            category: 'Banana',
+            keybindings: [{ keys: ['Ctrl+B', 'B'], isDefault: false }],
+            args: undefined
+          },
+          {
+            id: '4',
+            label: 'Tomato',
+            command: 'Tomato',
+            selector: 'Tomato',
+            category: 'Tomato',
+            keybindings: [{ keys: ['Tab+T', 'T'], isDefault: true }],
+            args: undefined
+          }
+        ];
+      });
+
+      it('should test sortBy `category` column', () => {
```

Comment:
Tiny suggestion to use plain language in test descriptions to make them easier to read.

```suggestion
      it('should sort by the `category` column', () => {
```